src/components/SearchBar.tsx

```diff
@@ -0,0 +1,47 @@
+import './SearchBar.css';
+import searchIcon from '../assets/searchIcon.svg';
+import closeIcon from '../assets/close.svg';
+
+interface SearchBarProps {
+  setSearchQuery: (value: string) => void;

```

Comment:
```suggestion
setSearchQuery: Dispatch<React.SetStateAction<string>>;
```

권고사항: setState를 하는 함수와 그냥 함수가 다른 점이 있어서 저는 구분지어 타입을 지정해주는 편입니다. 
저는 상태관리하는 함수와, 데이터를 다루는 함수를 타입상으로 구분지어줌으로서 심리적으로 구분을 지어주려고 합니다.